sklearn/feature_selection/rfe.py

```diff
@@ -377,23 +395,34 @@ def fit(self, X, y):
         scorer = check_scoring(self.estimator, scoring=self.scoring)
         n_features = X.shape[1]
         n_features_to_select = 1
+        rfe = RFE(estimator=self.estimator,
+                  n_features_to_select=n_features_to_select,
+                  step=self.step, verbose=self.verbose - 1)
+
 
-        # Determine the number of subsets of features
-        scores = []
+        # Determine the number of subsets of features by fitting across
+        # the train folds and choosing the "features_to_select" parameter
+        # that gives the least averaged error across all folds.
 
-        # Cross-validation
-        for n, (train, test) in enumerate(cv.split(X, y)):
-            X_train, y_train = _safe_split(self.estimator, X, y, train)
-            X_test, y_test = _safe_split(self.estimator, X, y, test, train)
+        # Note that joblib raises a non-picklable error for bound methods
+        # even if n_jobs is set to 1 with the default multiprocessing
+        # backend.
+        # Passing bound methods as scorer objects to RFECV was possible
+        # before version 0.18. This branching is done so that to
+        # make sure that user code that sets n_jobs to 1
+        # and provides bound methods as scorers is not broken with the
+        # addition of n_jobs parameter.
```

Comment:
If passing bound methods as scorer objects to RFECV is deprecated, you can add a comment to know when this workaround can be removed.